testing/testcafe/model/contextMenu/index.ts

```diff
@@ -41,4 +50,9 @@ export default class ContextMenu extends Widget {
       { dependencies: { getInstance } },
     )();
   }
+
+  findItemByText(text: string): Selector {
+    const span = this.element.find(`.${CLASS.itemText}`);
+    return span.withExactText(text).parent();
+  }
```

Comment:
Why do we implement this as a separate method?

- Maybe we can move this code to the ```getItemByText```?
- If we prefer to have this logic as a separate method, let's add ```private``` to it.